roles/ci_multus/molecule/default/converge.yml

```diff
@@ -22,6 +22,7 @@
     cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
     cifmw_openshift_kubeconfig: "{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig"
     testpod_name: "pod-testnad"
+    cifmw_ci_multus_namespace: "ci-multus"
```

Comment:
Is this needed? Why? Or better said, why moving into a collection needs this.

Reply:
I needed it to run the molecule tests locally, but it was working fine in CI without it, I must have added it by mistake